src/main/java/com/puppycrawl/tools/checkstyle/AbstractAutomaticBean.java

```diff
@@ -313,6 +318,24 @@ public Object convert(Class type, Object value) {
 
     }
 
+    /** A converter that converts a string array to a pattern. */
```

Comment:
> that converts a string array 

Isn't it just converting a string and not a string array?

Reply:
done